app/mailers/lead_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class LeadMailer < ApplicationMailer
+  def new_lead(lead)
+    @lead = lead
+    
+    mail(
+      to: "info@shovelsquad.com",
+      from: "info@shovelsquad.com",
+      subject: "Phone call requested #{@lead.phone_number}"
```

Comment:
Matter of personal taste, but I don't think the phone number should appear in the subject line

Reply:
i'm waiting to hear what client actually wants in this email, I just made sure all fields were showing somehwere